src/script/lua_api/l_util.cpp

```diff
@@ -499,6 +499,21 @@ int ModApiUtil::l_upgrade(lua_State *L)
 	return 1;
 }
 
+int ModApiUtil::l_get_secret_key(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+#ifdef __ANDROID__
```

Comment:
This will need changing to `#if defined(__ANDROID__) || defined(__IOS__)` once iOS support is added